src/main/java/com/google/sps/data/OrderHandler.java

```diff
@@ -152,16 +154,40 @@ Entity getClosestLibrary(List<Entity> libraries, Point address) throws ApiExcept
   }
 
   /**
-   * Get unassigned orders from datastore with the area set to area.
-   * TODO[ak47na]: change method to return a list of Order objects so that classes that call it 
-   * don't need to know how data is stored.
+   * Returns the keyString of unassigned orders from datastore with the area set to area.
    */
-  List<Entity> getAvailableOrders(int area) {
+  List<String> getAvailableOrders(int area) {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     Query orderQuery = new Query("Order")
         .setFilter(new Query.FilterPredicate(OrderProperty.AREA.label, Query.FilterOperator.EQUAL, area))
         .setFilter(new Query.FilterPredicate(OrderProperty.STATUS.label, Query.FilterOperator.EQUAL, OrderStatus.ADDED.toString()));
 
-    return datastore.prepare(orderQuery).asList(FetchOptions.Builder.withDefaults());
+    List<Entity> results = datastore.prepare(orderQuery).asList(FetchOptions.Builder.withDefaults());
+    List <String> resultsKeyStrings = new ArrayList<>();
+    for (Entity order : results) {
+      resultsKeyStrings.add(KeyFactory.keyToString(order.getKey()));
+    }
+    return resultsKeyStrings;
+  }
+
+  /** 
```

Comment:
New public methods call for new tests in OrderHandlerTest.java

Reply:
I added one test for it.